ui/src/app/app-routing.module.ts

```diff
@@ -391,16 +389,16 @@ const routes: Routes = [
             data: { title: 'Workspace Admin'}
           }, {
             path: 'institution',
-            component: AdminInstitutionComponent,
-            data: { title: 'Institution Admin'}
+            component: AppRouting,
+            data: {}
           }, {
             path: 'institution/add',
```

Comment:
why we still have `path` in two places? And which one is the source of truth now?

Reply:
We decided to replace the router piece by piece instead of removing the whole thing at once in order to not silo one developer on this project for an extended period of time. `app-routing.module.ts` is technically the source of truth. What it's saying here is that instead of the Angular router rendering the component directly for this route, it will render the React router, which will decide what component to render based on the route.

When all the routes are migrated, we can dismantle `app-routing.module.ts`.